### PR TITLE
Create globalbrain-node node extension for calling GlobalBrain.jl from javascript

### PR DESCRIPTION
**Pull request by johnwarden**

### CODE REVIEW

File: globalbrain-node/Manifest.toml.bak [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# This file is machine-generated - editing it directly is not advised
```
Comment (fdietze) [inline globalbrain-node/Manifest.toml.bak:1]: Is this a backup file (`.bak`)? Or is this actually used?

File: globalbrain-node/assets/binding.png
Comment (fdietze) [inline globalbrain-node/assets/binding.png:1]: what are the assets needed for?

File: globalbrain-node/julia/.DS_Store
Comment (fdietze) [inline globalbrain-node/julia/.DS_Store:1]: add this file to `.gitignore` instead

File: globalbrain-node/example/package.json
Comment (fdietze) [inline globalbrain-node/example/package.json:1]: as far as I understand, this example is not needed, right? There is a separate `test.js`, which is referenced in the `justfile`.

File: globalbrain-node/julia/.DS_Store
Comment (johnwarden) [reply globalbrain-node/julia/.DS_Store:1]: Removed this file

File: globalbrain-node/example/package.json
Comment (johnwarden) [reply globalbrain-node/example/package.json:1]: Removed this file

File: globalbrain-node/assets/binding.png
Comment (johnwarden) [reply globalbrain-node/assets/binding.png:1]: Removed these files

**Pull request merged**